apps/odyssey-stats/src/components/odyssey-query-memberships.ts

```diff
@@ -0,0 +1,50 @@
+import { useQuery } from '@tanstack/react-query';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import wpcom from 'calypso/lib/wp';
+import getDefaultQueryParams from 'calypso/my-sites/stats/hooks/default-query-params';
+import { MEMBERSHIPS_PRODUCTS_RECEIVE } from 'calypso/state/action-types';
+import { membershipProductFromApi } from 'calypso/state/data-layer/wpcom/sites/memberships';
+import 'calypso/state/memberships/init';
+import { SiteId } from 'calypso/types';
+
+function queryMemberships() {
+	return wpcom.req
+		.get( { path: '/memberships/products', apiNamespace: 'wpcom/v2', isLocalApiCall: true } )
```

Comment:
```suggestion
		.get( { path: '/memberships/products?type=all&is_editable=true', apiNamespace: 'wpcom/v2', isLocalApiCall: true } )
```
Do we need the same query arguments as [the direct API query](https://github.com/Automattic/wp-calypso/blob/33fadd8c1449f1ea4181b8359c4fc36b7a34bc18/client/state/data-layer/wpcom/sites/memberships/index.js#L62), which makes the returned products work well?

<img width="1101" alt="截圖 2024-06-10 上午12 24 16" src="https://github.com/Automattic/wp-calypso/assets/6869813/baa1e5e7-af39-4b9f-8b57-c668d9fca16e">

Reply:
Addressed in c4f19bbb6fd